src/settings/portalSettings.ts

```diff
@@ -1,29 +1,47 @@
+import {
+  faSun as solidSun,
+  faUserFriends as solidUsers,
+  IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { SPACE_TAXON } from "settings/taxonomy";
+
 import { RoomVisibility } from "types/venues";
 
-export type LabelVisibilityOption = {
+type VisibilityOption = {
   label: string;
-  value: string;
+  value: RoomVisibility;
+  subtitle?: { text: string; icon: IconDefinition }[];
 };
 
-export const LABEL_VISIBILITY_OPTIONS: LabelVisibilityOption[] = [
-  {
-    label: "Always",
-    value: RoomVisibility.nameCount,
+export type LabelVisibilityOptions = {
+  noLabel: VisibilityOption;
+  hoverTitle: VisibilityOption;
+  countOnly: VisibilityOption;
+  titleAndCount: VisibilityOption;
+};
+
+export const LABEL_VISIBILITY_OPTIONS: LabelVisibilityOptions = {
```

Comment:
Having `Record` as a type makes the structure more robust to change and easier to maintain

```suggestion
export const LABEL_VISIBILITY_OPTIONS:  Record<RoomVisibility,VisibilityOption>= {
  [RoomVisibility.none]: {
     // etc.
  },
  // etc.
```

Reply:
self-resolved by #2523